report/telegram.go

```diff
@@ -0,0 +1,73 @@
+package report
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/future-architect/vuls/config"
+	"github.com/future-architect/vuls/models"
+)
+
+// ChatWorkWriter send report to ChatWork
```

Comment:
__[Golint]__ comment on exported type TelegramWriter should be of the form "TelegramWriter ..." (with optional leading article)

<sub>posted by Sider</sub>